eos/form-factors/parametric-kkvdz2022_TEST.cc

```diff
@@ -0,0 +1,87 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Danny van Dyk
+ * Copyright (c) 2022 Stephan Kürten
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <test/test.hh>
+#include <eos/form-factors/parametric-kkvdz2022.hh>
+
+using namespace test;
+using namespace eos;
+
+class ParametricKKvDZ2022FormFactorTest :
+    public TestCase
+{
+    public:
+        ParametricKKvDZ2022FormFactorTest() :
+            TestCase("parametric_kkvdz2022_form_factor_test")
+        {
+        }
+
+        virtual void run() const
+        {
+            static const double eps = 1.0e-5;
+
+            Parameters p = Parameters::Defaults();
+
+            {
+                KKvDZ2022FormFactors kkvdz2022(p, Options{ });
+
+                //The form factors are functions of F_i(q2,k2)
```

Comment:
That comment makes no sense. Also ``// A space is needed after a comment begins``.

Reply:
Please add a comment on against what implementation you tested. Your own? Marvin's?